spec/index.bs

```diff
@@ -1175,14 +1175,24 @@ This document modifies {{HTMLScriptElement}}s. Each script has:
     through a compliant sink. Equivalent to script's
     [=child text content=]. Initially an empty string.
 
+This document also modifies {{SVGScriptElement}}s. Each script has:
+
+: an associated string <dfn export for="SVGScriptElement">script text</dfn>.
+::  A string, containing the body of the script to execute that was set
+    through a compliant sink. Equivalent to script's
+    [=child text content=]. Initially an empty string.
+
+Issue: There's currently no defined processing model for {{SVGScriptElement}}s. This spec assumes one that is similar to that of {{HTMLScriptElement}}'s model.
```

Comment:
This feels iffy but there really seems to be no spec for SVGScriptElement's atm. So this is an attempt to say "use common sense and good luck"

Reply:
There's https://www.w3.org/TR/SVGMobile12/script.html#ScriptContentProcessing which is an old version of an SVG spec which could be monkey patched but idk that it really provides much value doing that.